components/filter/filter.js

```diff
@@ -882,6 +938,13 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		});
 	}
 
+	_setSelectedOnRender(dimension) {
```

Comment:
I'm wondering if `setInitiallySelected` and `initiallySelected` is a little clearer? If that's not accurate, then I still don't know what `selectedOnRender` means lol.  Or maybe `saveCurrentlySelected` and `currentlySelected`?

Reply:
Maybe `currentlySelected` could work but it's not exactly accurate since it saves the state it was selected. It used to be `selectedOnOpen` but after changes that was no longer accurate. The property will save whether a value was selected when first opening the dimension and when searching. 

We didn't want the selected items to bubble to the top when you click on them so it only recalculates when you re-render the full list due to searching.

Reply:
`shouldBubble` and `_updateSetDimensionShouldBubble`? Or maybe that's weird lol

Reply:
I like that better! Decouples the logic from the name so if we change up when we bubble in the future, the variable name is still accurate. Maybe just `_updateDimensionShouldBubble` to shorten it, similar to `_isDimensionEmpty()` or `_buildDimension()`